src/scriptworker/constants.py

```diff
@@ -512,7 +512,7 @@
                     "firefox": "GECKO",
                     "thunderbird": "COMM",
                     "mobile": "MOBILE",
-                    "mpd001": "MPD001",
+                    "mozillavpn": "MOZILLAVPN",
```

Comment:
This means we're going to look for `MOZILLAVPN_HEAD_REV` etc, which will be relevant in your `.tc.yml`